cmd/ddev/cmd/addon-list.go

```diff
@@ -133,12 +132,9 @@ func renderRepositoryList(repos []*github.Repository) string {
 }
 
 func init() {
-	AddonListCmd.Flags().Bool("all", false, `List unofficial DDEV add-ons for in addition to the official ones`)
```

Comment:
Let's make it hidden instead of removing it.

We can list it for removal in future releases.

BTW, we still have this:
```
ddev get --list --all
```